cfgov/ask_cfpb/models/pages.py

```diff
@@ -340,7 +340,12 @@ def portal_category_page(self, request, **kwargs):
             portal_topics=self.portal_topic.heading,
             language=self.language,
             portal_categories=self.portal_category.heading)
-        return self.get_results(request)
+        # call results here to activate translations, to get the right title
```

Comment:
Why do translations need to be activated?

Reply:
Ah great point. I was initially including "answers" in the category title, which needed to be translated, but we just decided to ditch that, so we don't need to activate early.

Reply:
Early `get_results` invocation removed in 96445fc